src/CoreBundle/private/modules/CMSTableExport/CMSTableExport.class.php

```diff
@@ -145,6 +149,28 @@ public function GenerateExport()
         $this->getDownload($sFileType);
     }
 
+    private function provideExportMemory(): void
+    {
+        $exportMemoryUnits = ServiceLocator::getParameter('chameleon_system.core.export_memory');
+
+        $inBytes = $this->unitToInt($exportMemoryUnits);
+
+        if ($inBytes > 256*1024*1024) {
```

Comment:
what is this for? I assume you want to set it only if it exceeds the current memory limit?

Reply:
not exactly; simply filter out "bad" values ie "4M"

Reply:
But I could of course simply trust the user (admin)?

(And get rid of the nice unitToInt method...)

Reply:
I think that makes sense. If you do want to validate the input, then the compile Configuration class would be the place to do so.